shub/apps/main/views/download.py

```diff
@@ -141,6 +141,4 @@ def _download_container(container, request):
 
         # There is no container
         raise Http404
```

Comment:
We probably don't need two here :) Did you run some automated linter to make these changes?